src/features/cookie_consent_banner/components/CookieConsentBanner.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button, Card, Heading, Animation } from '@konturio/ui-kit';
+import { useAtom } from '@reatom/react';
+import ReactMarkdown from 'react-markdown';
+import { userResourceAtom } from '~core/auth';
+import { AppFeature } from '~core/auth/types';
+import { i18n } from '~core/localization';
+import { cookieSettingsAtom } from '../atoms/cookieSettingsAtom';
+import s from './CookieConsentBanner.module.css';
+
+export function CookieConsentBanner() {
+  const [havePrompts, { acceptAll, rejectAll }] = useAtom(cookieSettingsAtom);
```

Comment:
shouldn't this atom run in forseRun and Banner be like ?

```
      <Suspense fallback={null}>
        {userModel?.hasFeature(AppFeature. USE_3RDPARTY_ANALYTICS) && < CookieConsentBanner />}
      </Suspense>
```

Reply:
@propakov thank you, fixedl